scripts/install/workers.sh

```diff
@@ -39,4 +39,11 @@ do
       fi
 
     fi
+
+if [[ ! -e ~/nltk_wordlist_installations/stopwords.log ]]; then
```

Comment:
This should not be looking for the existence of a log file, that is not a stable test. It should be checking that the installed directory is not empty; this can be done like so:
```bash
#!/bin/bash
FILE=""
DIR="$HOME/.augur/nltk-wordlists"
if [ "$(ls -A $DIR)" ]; then
 echo "Take action $DIR is not Empty"
else
 echo "$DIR is Empty"
fi
```